charmcraft.yaml

```diff
@@ -92,4 +90,20 @@ requires:
   ingress:
     interface: traefik_route
     limit: 1
-
+config:
+  options:
+    custom-config-repository:
+      type: string
+      description: >
+        The repository URL where the Wazuh configuration is stored. The username has to be provided as in 
+        git+ssh://username@repository@branch, where the branch is optional.
```

Comment:
Out of curiosity, do all configuration repositories have to be public?

Reply:
No, that's why you can specify a private key to authenticate